tests/ut/azure_iot_json_writer_ut.c

```diff
@@ -592,6 +592,60 @@ static void testAzureIoTJSONWriter_AppendArray_Success( void ** ppvState )
     assert_string_equal( ucJSONWriterBuffer, ucTestJSONArray );
 }
 
+static void testAzureIoTJSONWriter_InvalidWrite_Failure( void ** ppvState )
+{
+    AzureIoTJSONWriter_t xWriter;
+    uint8_t ucBuffer[ 5 ];
+
+    assert_int_equal( AzureIoTJSONWriter_Init( &xWriter, ucBuffer, sizeof( ucBuffer ) ), eAzureIoTSuccess );
+
+    /*Begin Object */
+    assert_int_equal( AzureIoTJSONWriter_AppendBeginObject( &xWriter ), eAzureIoTSuccess );
+
+    /*Fail PropertyName value api's */
```

Comment:
nit: since I can't see what `ucProperty` is local here, maybe add a comment as to why this fails? Took me a second to figure out.
```suggestion
    /*Fail PropertyName value api's (not enough space in buffer) */
```